traffic_ops/traffic_ops_golang/server/servers.go

```diff
@@ -1630,39 +1537,66 @@ func Update(w http.ResponseWriter, r *http.Request) {
 		}
 	}
 
-	changeLogMsg := fmt.Sprintf("SERVER: %s.%s, ID: %d, ACTION: updated", *server.HostName, *server.DomainName, *server.ID)
+	changeLogMsg := fmt.Sprintf("SERVER: %s.%s, ID: %d, ACTION: updated", *srvr.HostName, *srvr.DomainName, *srvr.ID)
 	api.CreateChangeLogRawTx(api.ApiChange, changeLogMsg, inf.User, tx)
 }
 
+func updateServer(tx *sqlx.Tx, server tc.ServerV40) (int64, int, error, error) {
+
+	rows, err := tx.NamedQuery(updateQuery, server)
+	if err != nil {
+		userErr, sysErr, errCode := api.ParseDBError(err)
+		return 0, errCode, userErr, sysErr
+	}
+	defer rows.Close()
+
+	var serverId int64
+	rowsAffected := 0
+	for rows.Next() {
+		if err := rows.Scan(&serverId); err != nil {
+			return 0, http.StatusNotFound, nil, fmt.Errorf("scanning lastUpdated from server insert: %v", err)
+		}
+		rowsAffected++
+	}
+
+	if rowsAffected < 1 {
+		return 0, http.StatusNotFound, errors.New("no server found with this id"), nil
+	}
+	if rowsAffected > 1 {
+		return 0, http.StatusInternalServerError, nil, fmt.Errorf("update for server #%d affected too many rows (%d)", *server.ID, rowsAffected)
+	}
+
+	return serverId, http.StatusAccepted, nil, nil
+}
+
 func createV2(inf *api.APIInfo, w http.ResponseWriter, r *http.Request) {
 	var server tc.ServerNullableV2
 
-	tx := inf.Tx.Tx
```

Comment:
We seem to do a mix of `tx` and `inf.Tx.Tx` throughout this file and other various handlers elsewhere. I see `inf.Tx.Tx` being used more frequently throughout the codebase, so I removed this since it seemed unnecessary and I preferred the consistency

Reply:
Honestly, we should probably make it consistent the _other_ way and start using just `tx` in most places. I think this was added here because it's annoying to have to type `inf.Tx.Tx` everywhere and remembering that it's not inf.Tx but `inf.Tx.Tx` in most places. That said, I don't really care to revert this change